setupMFEM.sh

```diff
@@ -0,0 +1,5 @@
+make config MFEM_CXX="nvcc" \
```

Comment:
This is certainly convenient, but should be incorporated into the MFEM makefiles (and maybe CMake? - @camierjs) before merging.

Reply:
Definitely! It was a cheat script for me. I wasn't sure on how things in the build system should be modified, but I can made adjustments accordingly.